liferay-workspace/modules/blade.indexerpostprocessor/src/main/resources/META-INF/module-log4j.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
Would you have time to submit a PR that adds logging to all of the modules in the same way?